Steer/DigitizerWorkflow/src/SimpleDigitizerWorkflow.cxx

```diff
@@ -157,6 +157,9 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
   workflowOptions.push_back(
     ConfigParamSpec{"configFile", VariantType::String, "", {"configuration file for configurable parameters"}});
 
+  // option to disable MC truth
+  workflowOptions.push_back(ConfigParamSpec{"no-mc-truth", o2::framework::VariantType::Bool, false, {"disable  mc-truth"}});
```

Comment:
somehow we need to synchronize, the tpc-reco-workflow is using `--disable-mc`

Reply:
If you want, but `disable-mc` sounds too vague to me.

Reply:
this was more some loud thinking, probably `no-mc-truth` is better and I can adjust the reco workflow, we should check if there are more places. Anyhow, no stopper for this PR

Reply:
all reco workflows have an option ``--disable-mc``, I would keep it.